src/package.json

```diff
@@ -22,5 +22,8 @@
     "sass-loader": "^12.6.0",
     "webpack": "^5.96.1",
     "webpack-stream": "^7.0.0"
+  },
+  "overrides": {
+    "semver": "^7.5.3"
```

Comment:
Everything works fine so probably not a big deal, but is there a reason we're pinning the semver version here?

Reply:
Good question and copying over our slack thread for doc's sake

there’s unfortunately no compatible patched version available for the stable semver version 7.5.2 since pa11y sets its own semver req of 7.3.8. Since the insecure version is coming from a dependency of a dependency rather than our own dependency, we ended up marking it ourselves in override. Ideally pa11y (and I believe another ci dependency that has the same issue) would patch it on their end eventually though